includes/external_repositories/external_repository_pft.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+if [ "$FILECHECKER_SH" == "1" ]
+then
+
+  function pft_append_disabled_tests_list
+  {
+	echo "\n" >> .mypft
+	echo "==========================" >> .mypft
+	echo "  Disabled Tests" >> .mypft
+	echo "==========================" >> .mypft
+	echo "" >> .mypft
+	${PFT_DIR}"/show-disabled-tests" >> .mypft
+  }
+
+  function check_pft
+  {
+    local RET0 RET1 TOTAL MYPATH PASSED TOTAL
+    if [ "$OPT_NO_PFT" == "0" ] ; then
+      rm -f .mypft
+      if ! [ -d "${PFT_DIR}" ] ; then
+        printf "%s" "Not installed"
+        return 1
+      fi
+      MYPATH=$(get_config "ft_printf")
+      make -C "${PFT_DIR}" 1> .mypft 2>&1
+      # Check compilation
+      RET1=`cat .mypft`
+      RET0=`echo "$RET1" | grep "error"`
+      if ! [ "$RET0" == "" ] ; then
+        printf "%s" "Cannot compile"
+        return 1
+      fi
+      # Run tests
+      "${PFT_DIR}/test" 1> .tmp1 2>&1
```

Comment:
could you please use more specific filename for temporary logs: for example `.mypft_tmp1`

Reply:
should be resolved in latest commit